internal/handlers/common/aggregations/operators/operators.go

```diff
@@ -13,27 +13,96 @@
 // limitations under the License.
 
 // Package operators provides aggregation operators.
+// Operators are used in aggregation stages to filter and model data.
+// This package contains all operators apart from the accumulation operators,
+// which are stored and described in accumulators package.
+//
+// Accumulators that can be used outside of accumulation with different behaviour (like `$sum`),
+// should be stored in both operators and accumulators packages.
 package operators
 
-import (
-	"context"
+import "github.com/FerretDB/FerretDB/internal/types"
+
+// newOperatorFunc is a type for a function that creates a standard aggregation operator.
+//
+// By standard aggregation operator we mean any operator that is not accumulator.
+// While accumulators perform operations on multiple documents
+// (for example `$count` can count documents in each `$group` group),
+// standard operators perfrom operations on single documents.
+type newOperatorFunc func(expression *types.Document) (Operator, error)
+
+// Operator is a common interface for standard aggregation operators.
+type Operator interface {
+	// Process document and returns the result of applying operator.
+	Process(in *types.Document) (any, error)
```

Comment:
If `Accumulator` interface has method `Accumulate`, `Operator` interface should have method `Operate`! :D 

(minor semi-joke comment)